src/components/AddRestaurantModal.jsx

```diff
@@ -1,48 +1,78 @@
-          <h2 className="modal-title text-title">새로운 음식점</h2>
-          <form>
-            {/* 카테고리 */}
-            <div className="form-item form-item--required">
-              <label htmlFor="category text-caption">카테고리</label>
-              <select name="category" id="category" required>
-                <option value="">선택해 주세요</option>
-                <option value="한식">한식</option>
-                <option value="중식">중식</option>
-                <option value="일식">일식</option>
-                <option value="양식">양식</option>
-                <option value="아시안">아시안</option>
-                <option value="기타">기타</option>
-              </select>
-            </div>
-
-            {/* 음식점 이름 */}
-            <div className="form-item form-item--required">
-              <label htmlFor="name text-caption">이름</label>
-              <input type="text" name="name" id="name" required />
-            </div>
-
-            {/* 설명 */}
-            <div className="form-item">
-              <label htmlFor="description text-caption">설명</label>
-              <textarea name="description" id="description" cols="30" rows="5"></textarea>
-              <span className="help-text text-caption">메뉴 등 추가 정보를 입력해 주세요.</span>
-            </div>
-
-            {/* 추가 버튼 */}
-            <div className="button-container">
-              <button className="button button--primary text-caption">추가하기</button>
-            </div>
-          </form>
-        </div>
+    <div className="modal modal--open">
+      <div className="modal-backdrop" onClick={() => setIsAddModalOpen(false)}></div>
+      <div className="modal-container">
+        <h2 className="modal-title text-title">새로운 음식점</h2>
+        <form>
+          <div className="form-item form-item--required">
+            <label htmlFor="category text-caption">카테고리</label>
+            <select
+              name="category"
+              id="category"
+              required
+              onChange={(selected) => setSelectedCategory(selected.target.value)}
+            >
+              <option value="">선택해 주세요</option>
+              {CATEGORY_DATA.slice(1).map((category) => (
+                <option key={category}>{category}</option>
+              ))}
+            </select>
+          </div>
+
+          <div className="form-item form-item--required">
+            <label htmlFor="name text-caption">이름</label>
+            <input
+              type="text"
+              name="name"
+              id="name"
+              required
+              onChange={(input) => setRestaurantName(input.target.value)}
+            />
+          </div>
+
+          <div className="form-item">
+            <label htmlFor="description text-caption">설명</label>
+            <textarea
+              name="description"
+              id="description"
+              cols="30"
+              rows="5"
+              onChange={(input) => setRestaurantInfo(input.target.value)}
+            ></textarea>
```

Comment:
```suggestion
            <textarea
              name="description"
              id="description"
              cols="30"
              rows="5"
              value={restaurantInfo}
              onChange={(input) => setRestaurantInfo(input.target.value)}
            ></textarea>
```

React에 의해 제어되도록 제어 컴포넌트를 사용하시기로 결정하셨다면, 이 요소의 `value` 또한 React에서 내려주는 state를 사용하실 것을 강력히 권장해 드리고 싶어요. 아래에 이유를 적어볼게요 ✏️ 

- `onChange` 이벤트가 발생하는 경우 `restaurantInfo` state가 업데이트되므로 당장은 문제가 없는 것처럼 보일 수 있어요.
- 그러나, `restaurantInfo` state가 다른 로직에 의해 업데이트되는 일이 생길 때에는 이 `<textarea>` 컴포넌트는 `value` 값이 React의 state로 되어 있지 않기 때문에 React의 최신화된 state 값이 반영되지 않아요. "다른 로직에 의해 업데이트 되는 일" 을 아래에 조금 설명해 볼게요.
    - 이후 [지우기] 버튼을 추가해 `restaurantInfo`를 포함한 모든 정보를 지우는 로직을 추가한다면?
    - 몇몇 폼들은 하나의 인풋이 변경되었을 때 여러 개의 인풋의 정보가 자동으로 바뀌도록 하거나, 보이는 메뉴를 다르게 하기도 하는 기능을 구현한다면?
- 즉, 이러한 작업을 해 주는 이유는 이후 state가 변화하더라도 정확한 값을 지닐 수 있도록 **동기화** 해 주기 위함이에요.

Reply:
오호... 리액트에서 상태는 생각보다 엄청 중요한 개념이네요. 

> React에 의해 제어되도록 제어 컴포넌트를 사용하시기로 결정하셨다면, 이 요소의 value 또한 React에서 내려주는 state를 사용하실 것을 강력히 권장해 드리고 싶어요. 

명심하겠습니다!!